specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -477,6 +477,105 @@
         }
       }
     },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/customers": {
+      "get": {
+        "tags": [
+          "Customers"
+        ],
+        "x-ms-examples": {
+          "CustomersListByBillingAccountName": {
+            "$ref": "./examples/CustomersListByBillingAccountName.json"
+          }
+        },
+        "operationId": "Customers_ListByBillingAccountName",
+        "description": "Lists all customers which the current user can work with on-behalf of a partner.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/billingAccountNameParameter"
+          },
+          {
+            "name": "$filter",
+            "description": "May be used to filter using hasPermission('{permissionId}') to only return customers for which the caller has the specified permission.",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "$skiptoken",
+            "description": "Skiptoken is only used if a previous operation returned a partial result. If a previous response contains a nextLink element, the value of the nextLink element will include a skiptoken parameter that specifies a starting point to use for subsequent calls.",
```

Comment:
>nextLink [](start = 136, length = 8)

You need to add x-ms-pageable to match this as well.

Reply:
Done.